pkg/standalone/run.go

```diff
@@ -44,8 +50,8 @@ type RunConfig struct {
 	MetricsPort      int      `env:"DAPR_METRICS_PORT" arg:"metrics-port" yaml:"metrics_port"`
 	UnixDomainSocket string   `arg:"unix-domain-socket" yaml:"unix_domain_socket"`
 	InternalGRPCPort int      `arg:"dapr-internal-grpc-port" yaml:"dapr_internal_grpc_port"`
-	DaprPathCmdFlag  string   `yaml:"dapr_path_cmd_flag"`
 	SharedRunConfig  `yaml:",inline"`
```

Comment:
Quick question, should `SharedRunConfig` be embedded in the `RunConfig` instead of being a field?

Reply:
Ok never mind, missed it. @pravinpushkar can you please add this to the top of the struct, embedded values by convention appear on the top.